ConsoleApp3/ConsoleApp3/Program.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+
+namespace ConsoleApp
+{
+
+    public class Stack<T>
+    {
+        public T[] Stck { get; set; }
+        public int Top { get; set; }
+        public int Size { get; }
+
+        public Stack(int Size)
+        {
+            Stck = new T[Size]; 
+            this.Size = Stck.Length;
+        }
+
+        public void Push(T Element)
+        {
+
+            if (Top == Size)
+            {
+                throw new InvalidOperationException("You are nerd!!! STACK OVERFLOW!!!");
+            }
+            else
+            {
+            Stck[Top] = Element;
+            Top++;
+            }
+        }
+
+        public T Pop()
+        {
+            if (Top > 0)
+            {
+
+                T lastElement = Stck[Top-1];
+                Stck[Top - 1]=default(T);
+                Top--;
+                return lastElement;
+            }
+            return default(T);
```

Comment:
it's better to throw exception if there are no elements because it's possible that element equals to default